example/.env

```diff
@@ -0,0 +1,4 @@
+ELK_VERSION=7.15.2
+CACHE_ROOT=data/logs
+ELASTIC_PASSWORD=changeme
+ES_JAVA_OPTS=-Xmx1G -Xms256m
```

Comment:
perhaps mirror the comments on change your password and 16gb is recommended in here? Stick the elk version at the end

Reply:
✅